workload_generator/src/workload_gen.py

```diff
@@ -13,31 +13,38 @@
 import logging
 import queue
 import time
+#import requests
 from threading import Thread
 
 defaultTestFile = "../workloads/WL_1_USER.txt"
+
 serverAddress   = "localhost"
-serverPort      = 60000
-
-ADD = "ADD"
-QUOTE = "QUOTE"
-BUY = "BUY"
-COMMIT_BUY = "COMMIT_BUY"
-CANCEL_BUY = "CANCEL_BUY"
-SELL = "SELL"
-COMMIT_SELL = "COMMIT_SELL"
-CANCEL_SELL = "CANCEL_SELL"
-SET_BUY_AMOUNT = "SET_BUY_AMOUNT"
-CANCEL_SET_BUY = "CANCEL_SET_BUY"
-SET_BUY_TRIGGER = "SET_BUY_TRIGGER"
-SET_SELL_AMOUNT = "SET_SELL_AMOUNT"
-SET_SELL_TRIGGER = "SET_SELL_TRIGGER"
-CANCEL_SET_SELL = "CANCEL_SET_SELL"
-DUMPLOG = "DUMPLOG"
-DISPLAY_SUMMARY = "DISPLAY_SUMMARY"
+serverPort      = "8000"
+
+serverURL       = "http://" + serverAddress  + ":" + serverPort + "/"
+
+API_STOCKS_PATH     = "api/stocks/"
+API_ACCOUNTS_PATH   = "api/accounts/"
+API_TRIGGERS_PATH   = "api/triggers/"
+
+ADD                 = "ADD"
+QUOTE               = "QUOTE"
+BUY                 = "BUY"
+COMMIT_BUY          = "COMMIT_BUY"
+CANCEL_BUY          = "CANCEL_BUY"
+SELL                = "SELL"
+COMMIT_SELL         = "COMMIT_SELL"
+CANCEL_SELL         = "CANCEL_SELL"
+SET_BUY_AMOUNT      = "SET_BUY_AMOUNT"
+CANCEL_SET_BUY      = "CANCEL_SET_BUY"
+SET_BUY_TRIGGER     = "SET_BUY_TRIGGER"
+SET_SELL_AMOUNT     = "SET_SELL_AMOUNT"
+SET_SELL_TRIGGER    = "SET_SELL_TRIGGER"
+CANCEL_SET_SELL     = "CANCEL_SET_SELL"
+DUMPLOG             = "DUMPLOG"
+DISPLAY_SUMMARY     = "DISPLAY_SUMMARY"
 
 userQ = queue.Queue()
-packetQ = queue.Queue()
 
 class WorkloadGenerator:
     def __init__(self, testFile):
```

Comment:
Workload generator freezes/crashes for 1000+ user workload execution. 

I believe this is a result of the files it creates per user request. 

working on a fix for Sunday.

Reply:
Uh oh! Which files do you think? the individual user workload file?

Reply:
Uh oh! Which files do you think? the individual user workload file?